tasks/src/main/scala/dagr/tasks/bwa/Bwa.scala

```diff
@@ -61,7 +61,7 @@ object Bwa extends Configuration {
     val bwaBuffer  = new FifoBuffer[Sam].requires(memory=fifoMem)
     val altPipe    = if (doAlt) new BwaK8AltProcessor(altFile=alt) | new FifoBuffer[Sam].requires(memory=fifoMem) else Pipes.empty[Sam]
     val mergeBam   = new MergeBamAlignment(unmapped=unmappedBam, mapped=Io.StdIn, out=mappedBam, ref=ref, sortOrder=sortOrder, orientation=orientation)
-    mergeBam.requires(memory=Memory(mergeBamAlignmentMem))
+    mergeBam.requires(cores=Cores(0), memory=Memory(mergeBamAlignmentMem))
```

Comment:
Why zero cores for MergeBamAlignment?  I don't see the same for SamToFastq or the FifoBuffers.